product_docs/docs/pgd/4/rel_notes/pgd_4.2.1_rel_notes.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.2.1"
+navTitle: "Version 4.2.1"
+---
+
+EDB Postgres Distributed version 4.2.1 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+| Component | Version |  Type   |                                                                                                                                                                          Description                                                                                                                                     |
+| --------- | ------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.2.1   | Bug fix | Grant bdr.replication_status to bdr_read_all_stats (BDR-2523) <p>  Alllow the read_all_stats role to check BDR replication information. </p>                                                                                                                                                                             |
+| BDR       | 4.2.1   | Bug fix | Resolve hang related to `bdr.standby_slots_min_confirmed` (BDR-2445, RT80692) <p> Allow the WAl sender to proceed without the need to wait for the timeout after a Subscriber Only node is parted, while using standby_slot_names. </p>                                                                                  |
+| BDR       | 4.2.1   | Bug fix | Use two_phase slots in `bdr_init_physical` on PG14+ () <p> Improve prepared transaction handling during `bdr_init_physical`. </p>                                                                                                                                                                                        |
```

Comment:
```suggestion
| BDR       | 4.2.1   | Bug fix | Use two_phase slots in `bdr_init_physical` on PG14+ () <p> Improve prepared transaction handling during `bdr_init_physical`. </p>                                                                                                                                                                                        |
```
Is there a bug number for this? If not, let's remove the parens.

Reply:
updated the landing page for release notes. Please review and update the date and versions, if needed